code/__DEFINES/components.dm

```diff
@@ -205,6 +205,7 @@
 
 // /mob signals
 #define COMSIG_MOB_DEATH "mob_death"							//from base of mob/death(): (gibbed)
+#define COMSIG_MOB_REVIVE "mob_revive"							//from base of mob/on_revive(): ()
```

Comment:
excuse me this has one too many tabs it's out of line with the others

Reply:
that could just be github.

Reply:
It's github stuff for sure